cmd/queue/main.go

```diff
@@ -149,6 +149,10 @@ func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.H
 		switch {
 		case ph != "":
 			_, probeSpan := trace.StartSpan(r.Context(), "probe")
+			if ph == "ingress" {
```

Comment:
Why would ingress probe reach QP?

Reply:
Note the "Experimental" in the title :smile: 
It is not supposed to be ready to be reviewed.